traits/src/nft.rs

```diff
@@ -14,3 +17,20 @@ pub trait InspectExtended<AccountId>: Inspect<AccountId> {
 	/// Get the next token ID to be minted for a Class
 	fn next_token_id(class: Self::ClassId) -> Self::InstanceId;
 }
+
+// Supplement trait to the nonfungibles::Create trait
+pub trait CreateExtended<AccountId, ClassProperties>: Create<AccountId> {
+	/// The balance of account.
+	type Balance: AtLeast32BitUnsigned + FullCodec + Copy + MaybeSerializeDeserialize + Debug + Default;
+
+	// Returns the next class ID to be created.
+	fn next_class_id() -> Self::ClassId;
+	// Sets the property of the given class.
+	fn set_class_properties(class: &Self::ClassId, properties: ClassProperties) -> DispatchResult;
+	// Pays for the fee to mint tokens of a particular class.
+	fn pay_mint_fee(payer: &AccountId, class: &Self::ClassId, quantity: u32) -> DispatchResult;
```

Comment:
this just pay the fee?

I think this trait is very Acala nft specific so maybe just to in Acala repo instead of here

Reply:
ok i can move this trait into our acala